index.html

```diff
@@ -0,0 +1,192 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Bootstrap CSS -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+  <link href="css/main.css" rel="stylesheet">
+
+  <title>Google I/O</title>
+  
+</head>
+
+<body>
+  
+  <!-- top black pre-nav (not diplayed on desktop)-->
+  <div class="container-fluid black-prenav d-flex justify-content-end align-items-center">
+    <i class="fa-brands fa-facebook-f"></i>
+    <i class="fa-brands fa-twitter"></i>
+    <!-- <p>English</p> -->
+    <p>My I/O</p>
+    <p>Log Out</p>
+  </div>
+  
+  <div class="container-fluid nav-section">
+    <nav class="navbar navbar-expand-md navbar-light">
+      <div class="container-fluid nav-container">
+        <a class="navbar-brand logo" href="#">Google <span class="io">I/O</span></a>
+        <button class="navbar-toggler collapsed d-flex d-md-none flex-column justify-content-around" type="button"
+          data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="toggler-icon top-bar"></span>
+          <span class="toggler-icon middle-bar"></span>
+          <span class="toggler-icon bottom-bar"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav nav">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#" >Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Speakers</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </div>
+
+  <!-- hero section -->
+  <section class="container-fluid hero-section hero-background">
+    <div>
+      <h6>Hi! Google fans and welcome to</h6>
+      <h1>Google I/O Conference 2022</h1>
+      <p class="pr-3">
+        Connect with developers from around the world for thoughtful discussions,
+        learn from Google experts, and get a first
+        look at our latest developer products.
+      </p>
+      <h2 class="container">2022.05.11(WED) ~ 12(THUR)</h2>
+      <p class="container">@2000 N Shoreline Blvd Mountain View, CA 94043</p>
+    </div>
```

Comment:
Great job your design is nice. Since this is the first section that is being shown when opening your site, I'll recommend following the design and increasing the height. a `height: 100vh;` would have been better
**imp**
<img width="1426" alt="image" src="https://user-images.githubusercontent.com/37901469/169418746-14ce49cc-65af-488d-8c8c-dc36f6674def.png">
**Design**
<img width="974" alt="image" src="https://user-images.githubusercontent.com/37901469/169418808-311e5a7e-7a6f-4695-9f81-fff069e5e121.png">